util-taglib/src/META-INF/liferay-ui.tld

```diff
@@ -1972,6 +1972,13 @@
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<description>Sets the maximum number of items on the left box. The default value is <![CDATA[<code>-1</code>]]> that represents unlimited items.</description>
```

Comment:
Could I update these default values on .tld side?

Reply:
Nice catch, @diegonvs! Feel free to pull the changes in the branch and fix that so we can forward!

Thanks!